tests-e2e/cypress/e2e/edx-frontend-school/accessSchoolUsers.cy.ts

```diff
@@ -291,6 +291,51 @@ describe('Access School Users Page', () => {
       });
 
     });
+
+    context('temporary user to be deleted', () => {
+      let tempUserId = '';
+      let tempFirstName = '';
+      let tempUserId2 = '';
+      let tempFirstName2 = '';
+      let tempLastName2 = '';
+
+
+      before(() => {
+        cy.task('recreate-school', { schoolStatus: 'Open' });
+
+        cy.task<SchoolUserOptions, EdxUserEntity>('setup-schoolUser', { digitalId: crypto.randomUUID(), schoolCodes: ['99998']
+        }).then((user: EdxUserEntity) => { tempUserId = user.edxUserID; tempFirstName = user.firstName; });
+
+        cy.task<SchoolUserOptions, EdxUserEntity>('setup-schoolUser', { digitalId: crypto.randomUUID(), schoolCodes: ['99998']
+        }).then((user2: EdxUserEntity) => { tempUserId2 = user2.edxUserID; tempFirstName2 = user2.firstName; tempLastName2 = user2.lastName; });
+      });
+
+      beforeEach(() => {
+        cy.wrap(tempUserId).as('tempUserId');
+        cy.wrap(tempFirstName).as('tempUserFirstName');
+      });
+      // after(() => cy.logout());
+
+      it('can find delete option and cancel delete', () => {
+        navigateToAccessSchoolUsers();
+        cy.get('@tempUserId').then(uid => {
+        cy.get(`#user-remove-button-${uid}`).click();
+        cy.get(`#user-cancel-remove-button-${tempFirstName2}-${tempLastName2}`).click();
```

Comment:
This is probably broken.  It uses the UID from `@tempUserId` as the selector ID, which most likely refers to the first user, not the second.

Also, we should avoid using `tempFirstname2` and `tempLastName2` directly, as this breaks Cypress's isolation/concurrency model.  They should be wrapped first.